charts/longhorn/100.1.4+up1.2.6/values.yaml

```diff
@@ -0,0 +1,237 @@
+# Default values for longhorn.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+global:
+  cattle:
+    systemDefaultRegistry: ""
+    windowsCluster:
+      # Enable this to allow Longhorn to run on the Rancher deployed Windows cluster
+      enabled: false
+      # Tolerate Linux node taint
+      tolerations:
+      - key: "cattle.io/os"
+        value: "linux"
+        effect: "NoSchedule"
+        operator: "Equal"
+      # Select Linux nodes
+      nodeSelector:
+        kubernetes.io/os: "linux"
+      # Recognize toleration and node selector for Longhorn run-time created components
+      defaultSetting:
+        taintToleration: cattle.io/os=linux:NoSchedule
+        systemManagedComponentsNodeSelector: kubernetes.io/os:linux
+
+image:
+  longhorn:
+    engine:
+      repository: longhornio/longhorn-engine
+      tag: v1.2.6
+    manager:
+      repository: longhornio/longhorn-manager
```

Comment:
These should be mirrored images

Reply:
I didn't add `value.yaml.patch` to replace it with mirror images, I have fixed it.

Please help to review it again. Thank you.

Reply:
Hi Ray. this is still not fixed yet. The output here should contain mirrored images values